Resources/UIResources/uiPopupResources.robot

```diff
@@ -17,68 +17,147 @@ Resource    ${EXECDIR}/Inputs/testInputs.robot
 popupTestSetup
     [Documentation]    As a setup read all popup titles and messages/popup contents through excel sheet
     log to console    !-----Reading the inputs from the excel and storing in dictionary------!
-    testInputs.readingInputsFromExcel  popuptitlesmessages  A  B
+    testInputs.readingInputsFromExcel  uitest  A  B
     log to console    !-----PreCondition for the popup verification------!
 
 suppressAlarmPopup
     startBrowserAndLoginToAIEngineVX
```

Comment:
startBrowserAndLoginToAIEngineVX use uiresources.startBrowserAndLoginToAIEngineVX  for better readability.
Follow the same for all the keyword that are been called